docs/jwst/refpix/description.rst

```diff
@@ -28,44 +28,104 @@ detectors.
 Input details
 -------------
 
-The input file must be a ramp, and it should contain both a science
-('SCI') extension and a data quality ('DQ') extension.  The latter
-extension is normally added by the dq_init step, so running this
-step is a prerequisite for the refpix step.
+The input file must be a 4-D ramp and it should contain both a science
+(SCI) extension and a pixel data quality (PIXELDQ) extension. The PIXELDQ
+extension is normally populated by the ``dq_init`` step, so running that
+step is a prerequisite for the ``refpix`` step.
 
-Algorithm
----------
+Algorithms
+----------
 
-The algorithm for the NIR and MIR detectors is different.
-The entirely different algorithm for NIRSpec IRS2 readout mode is
+The algorithms for the NIR and MIR detectors are somewhat different.
+An entirely different algorithm for NIRSpec IRS2 readout mode is
 described in IRS2_.
 
 NIR Detector Data
 +++++++++++++++++
 
-#. The data from most detectors will have been rotated and/or flipped from their detector frame in order to give them the same orientation and parity in the telescope focal plane.  The first step is to transform them back to the detector frame so that all NIR and MIR detectors can be treated equivalently.
+#. The data from most detectors will have been rotated and/or flipped from
+   their detector frame in order to give them the same orientation and parity
+   in the telescope focal plane.  The first step is to transform them back to
+   the detector frame so that all NIR and MIR detectors can be treated equivalently.
 #. It is assumed that a superbias correction has been performed.
 #. For each integration, and for each group:
-    #. Calculate the mean value in the top and bottom reference pixels.  The reference pixel means for each amplifier are calculated separately, and the top and bottom means are calculated separately.  Optionally, the user can choose to calculate the means of odd and even columns separately by using the ``--odd_even_columns`` runtime parameter, as evidence has been found that there is a significant odd-even column effect in some datasets.  Bad pixels (those whose DQ flag has the DO_NOT_USE bit set) are not included in the calculation of the mean.
-    #. The mean is calculated as a clipped mean with a 3-sigma rejection threshold using the scipy.stats.sigmaclip method.
+    #. Calculate the mean value in the top and bottom reference pixels.
```

Comment:
The first line of this 3rd steps below, are being rendered oddly, probably because of the ":" at the end of the line.
To be clear, it is the "For each integration, and for each group:" line. Seems to be formatted as some type of note box.

Reply:
RST is really picky (just like TeX/LaTeX). That colon at the end of the line triggers a definition block, unless you have a blank line following it, in which case it comes out normal. Fixed.